Sources/Server/server.swift

```diff
@@ -1 +1,68 @@
+import Foundation
+import Socket
+import Requests
 
+public class Server {
+
+    var listeningSocket: Socket!
+    var clientSocket: Socket!
+    var readData = Data()
+    var parser: RequestParser
+
+    public init(parser: RequestParser) {
+        self.parser = parser
+    }
+
+    public func run() {
+        do {
+            try self.listeningSocket = Socket.create()
+            try self.listeningSocket.listen(on: 5000)
+            repeat {
+                try clientSocket = self.listeningSocket.acceptClientConnection()
+                let parsedIncomingRequest = parseRequest(socket: clientSocket)
+                let httpMethod = parsedIncomingRequest.returnMethod()
+
+                if (httpMethod == "GET" || httpMethod == "POST") {
```

Comment:
This is great considering the simplicity of the stories you worked on this week. This would probably be a great place to extract methods for a router in the future.